test/chi/components/data-table.pug

```diff
@@ -0,0 +1,1347 @@
+                  i.chi-icon.icon-plus(aria-hidden='true')
+            .chi-data-table__cell.-key.-bold
+              div Name 2
+            .chi-data-table__cell.-key
+              div name-2
+            .chi-data-table__cell
+              div 18 Dec 2020 2:38 a.m.
+          .chi-data-table__row.-collapsed.-xl
+            .chi-data-table__cell.-expandable
+              button.chi-button.-icon.-flat.-sm(aria-label='Expand row')
+                .chi-button__content
+                  i.chi-icon.icon-plus(aria-hidden='true')
+            .chi-data-table__cell.-key.-bold
+              div Name 3
+            .chi-data-table__cell.-key
+              div name-3
+            .chi-data-table__cell
+              div 5 Nov 2020 10:15 a.m.
+        .chi-data-table__footer
+          nav.chi-pagination.-compact(role='navigation' aria-label='Pagination')
+            .chi-pagination__content
+              .chi-pagination__start
+                .chi-pagination__results
+                  span.chi-pagination__label 6 Results
+              .chi-pagination__center
+                .chi-pagination__button-group.chi-button-group
+                  button.chi-button.-icon.-flat.-primary.-sm(aria-label='First page')
+                    .chi-button__content
+                      i.chi-icon.icon-page-first(aria-hidden='true')
+                  button.chi-button.-icon.-flat.-primary.-sm(aria-label='Previous page')
+                    .chi-button__content
+                      i.chi-icon.icon-chevron-left(aria-hidden='true')
+                .chi-pagination__label
+                  strong 1
+                  span of
+                  strong 2
+                .chi-pagination__button-group.chi-button-group
+                  button.chi-button.-icon.-flat.-primary.-sm(aria-label='Next page')
+                    .chi-button__content
+                      i.chi-icon.icon-chevron-right(aria-hidden='true')
+                  button.chi-button.-icon.-flat.-primary.-sm(aria-label='Last page')
+                    .chi-button__content
+                      i.chi-icon.icon-page-last(aria-hidden='true')
+              .chi-pagination__end
+
+.test-accordion-child
+  h2 Accordion child
+  .chi-card.-portal.-bg--white.-bg-md--grey-15
+    .chi-card__header.-sm
+      .chi-card__title Title
+    .chi-card__content.-p--0
+      .chi-data-table.-portal.-compact
+        .chi-data-table__head
+          .chi-data-table__row
+            .chi-data-table__cell.-expandable
+            .chi-data-table__cell
+              div Name
+            .chi-data-table__cell
+              div ID
+            .chi-data-table__cell
+              div Last Login
+        .chi-data-table__body
+          .chi-data-table__row.-expanded.-xl
+            .chi-data-table__cell.-expandable
+              button.chi-button.-icon.-flat.-sm(aria-label='Expand row')
+                .chi-button__content
+                  i.chi-icon.icon-minus(aria-hidden='true')
```

Comment:
This Expansion icon is from the Portal theme.
This and several following examples use portal markup. Please, keep the test page Lumen theme-specific.